docker-compose.yml

```diff
@@ -68,7 +68,8 @@ services:
     environment:
       - SMTP_USER=champlain.petclinic@gmail.com
       - SMTP_SERVER=smtp.gmail.com
-      - SMTP_PASS=nicetryguy
```

Comment:
not sure exactly why we would not have a pass so I'll leave a comment here and maybe receive some insight, not like I really fully understand what's going on here so that insight my just go through me.

Reply:
I removed it because it (should) have higher priority than the .env file where the real `SMTP_PASS` is loaded from